src/test/java/org/jabref/gui/preferences/keybindings/KeyBindingViewModelTest.java

```diff
@@ -0,0 +1,46 @@
+package org.jabref.gui.preferences.keybindings;
+
+import java.util.Optional;
+
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyEvent;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.keyboard.KeyBinding;
+import org.jabref.gui.keyboard.KeyBindingRepository;
+import org.jabref.preferences.PreferencesService;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+
+class KeyBindingViewModelTest {
+
+  @Test
+  void resetToDefault() {
+    // Set new key binding
+    KeyBindingRepository keyBindingRepository = new KeyBindingRepository();
+    KeyBindingsTabViewModel model = new KeyBindingsTabViewModel(keyBindingRepository, mock(DialogService.class), mock(PreferencesService.class));
```

Comment:
Var name is confusing, since this is a KeyBindings_TAB_ViewModel like KeyBindingViewModel later. Maybe better `tabViewModel`?